pennylane/templates/layers/strongly_entangling.py

```diff
@@ -43,7 +43,9 @@ def strongly_entangling_layer(weights, wires, r, imprimitive):
     n_wires = len(wires)
     if n_wires > 1:
         for i in range(n_wires):
-            imprimitive(wires=[wires[i], wires[(i + r) % n_wires]])
+            act_on = wires.subset([i, (i + r) % n_wires])
```

Comment:
(Minor suggestion), let me know if I missed something here
```suggestion
            act_on = wires.subset([i, i + r], periodic_boundary=True)
```

Reply:
great find!